api/src/main/java/org/pmiops/workbench/institution/InstitutionService.java

```diff
@@ -15,7 +16,8 @@
 
   Optional<Institution> getInstitution(final String shortName);
 
-  Optional<DbInstitution> getDbInstitution(final String shortName);
+  // throws NotFoundException if the DbInstitution is not found
+  DbInstitution getDbInstitutionOrThrow(final String shortName);
```

Comment:
Open to naming suggestions

Reply:
looks more like findDBInstitutionElseThrow but i know our convention is get so we can keep it as is